test/test_torch.py

```diff
@@ -3280,6 +3280,33 @@ def make_prob_dist(shape, is_contiguous):
     def test_multinomial(self):
         self._test_multinomial(self, torch.FloatTensor)
 
+    @staticmethod
+    def _test_multinomial_alias(self, cast):
+        # Get probs vector to use in setup
+        def get_probs(length, is_contiguous):
+            probs = torch.softmax(torch.randn(length), 0)
+            if not is_contiguous:
+                probs = torch.softmax(torch.randn(length, 2), 0)[:, 1]
+            assert not (is_contiguous ^ probs.is_contiguous()), "contiguity requirement not met"
+            return cast(probs)
+
+        for is_contiguous in [True, False]:
+            probs = get_probs(4, is_contiguous)
+            prob_table, alias_table = torch.multinomial_alias_setup(probs)
```

Comment:
What would happen if probs is n-dimensional (n > 1)? Would this throw an exception?